cEP-0005.md

```diff
@@ -0,0 +1,157 @@
+# coala Configuration
```

Comment:
Since we're talking about coafile reformatting, has a JSON coafile ever been brought up? I vaguely remember something related to JSON being discussed a few months ago (not sure if it was about coafiles, however). It doesn't offer anything more than the ini format, but it's more popular - see package.json.

There are some clear disadvantages too - comments aren't possible and it's not really designed for config files.

IMO ini files are better, but I just wanted to have a discussion on this, if that makes sense :)